.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Darkshore.lua

```diff
@@ -630,6 +663,13 @@ root(ROOTS.Zones, m(KALIMDOR, {
 					["timeline"] = { ADDED_4_0_3 },
 					["races"] = ALLIANCE_ONLY,
 					["groups"] = {
+						objective(1, {	-- 0/1 Use the Aetherion Ritual Orb
+							["provider"] = { "o", 194145 },	-- Aetherion Ritual Orb
```

Comment:
For objects, our tooltip system finds the 'closest' object by a given name, so a coordinate is required here